pkg/portfwd/listener.go

```diff
@@ -90,6 +105,9 @@ func (p *ClosableListeners) forwardTCP(ctx context.Context, client *guestagentcl
 	for {
 		conn, err := tcpLis.Accept()
 		if err != nil {
+			if opErr, ok := err.(*net.OpError); ok && opErr.Err.Error() == "use of closed network connection" {
```

Comment:
This should be changed to use `if errors.Is(err, net.ErrClosed) {`
opened #4048